independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2456 @@
+{

```

Comment:
@paulculmsee Please remove the other files you have added, i.e. "Screeenshot1.jpg" and "Screeenshot2.jpg" from the PR which have no references in the readme.md file.

Reply:
To certify the intent was to add screenshots which are referred to in the PR. Is the intent to host these elsewhere?

Reply:
@paulculmsee Sorry for the delay in response, You need to add them in the PR description, not in the connector folder. Please remove the screenshots to get this PR merged.

Reply:
Resolved with a previous commit